website/docs/d/log_analytics_workspace.html.markdown

```diff
@@ -46,6 +46,8 @@ The following attributes are exported:
 
 * `retention_in_days` - The workspace data retention in days.
 
+* `daily_quota_gb` - The workspace of the volume cap in gb.  Defaults to -1 (unlimited).
```

Comment:
Can we adjust the statement a bit? And no need to mention the default value for data source.

```suggestion
* `daily_quota_gb` - The workspace daily quota for ingestion in GB.
```